tables/he-common.uti

```diff
@@ -0,0 +1,581 @@
+always שֵּׂ 5-156-34  # sin with tsere
+always שֵׁ 146-34  # shin with dot
+always שֵּׁ 5-146-34  # shin with dot
+always שֵׂ 156-34  # sin
+always שֵּׂ 5-156-34  # sin with tsere
+always שֶּׁ 5-146-15  # shin with segol
+always שֶּׂ 5-156-15  # sin with segol
+always שֶׁ 146-15  # shin with dot
+always שֶּׁ 5-146-15  # shin with dot
+always שֶׂ 156-15  # sin
+always שֶּׂ 5-156-15  # sin with segol
+always שַּׁ 5-146-14  # shin with patah
+always שַּׂ 5-156-14  # sin with patah
+always שַׁ 146-14  # shin with dot
+always שַּׁ 5-146-14  # shin with dot
+always שַׂ 156-14  # sin
+always שַּׂ 5-156-14  # sin with patah
+always שָּׁ 5-146-126  # shin  qamats
+always שָּׂ 5-156-126  # sin  qamats
+always שָׁ 146-126  # shin with dot and qamats
+always שָּׁ 5-146-126  # shin with dot and qamats
+always שָׂ 156-126  # sin with qamats
+always שָּׂ 5-156-126  # sin  qamats
+always שְּׁ 5-146-3  # Shin with dot dagesh and sheva
+always שְּׂ 5-156-3  #  Sin with dagesh and sheva
+always שּׁ 5-146  # shin with dot and dagesh
+always שּׂ 156  # sin with dagesh
+always שׁ 146  # shin with dot
+always שְׁ 146-3  #  Shin with dot and sheva
+always שּׁ 5-146  # shin with dot and dagesh
+always שׂ 156  # sin
+always ת 1456  # tav
+always תְ 1256-3  # tav with dagesh sheva
+always תֱּ 1256-26  # tav dagesh with hataf segol
+always תֲּ 1256-25  # tav dagesh with hataf patah
+always תֳּ 1256-345  # tav dagesh with hataf qamats
+always תִּ 1256-24  # tav  dagesh with hiriq
+always תֵּ 1256-34  # tav with dagesh
+always תֶּ 1256-15  # tav with dagesh
+always תַּ 1256-14  # tav with dagesh
+always תָּ 1256-126  # tav dagesh with qamats
+always תּ 1256  # tav with dagesh
+always תְּ 1256-3  # tav with dagesh sheva
+always תֱּ 1256-26  # tav dagesh with hataf segol
+always תֲּ 1256-25  # tav dagesh with hataf patah
+always תֳּ 1256-345  # tav dagesh with hataf qamats
+always תִּ 1256-24  # tav  dagesh with hiriq
+always תֵּ 1256-34  # tav with dagesh
+always תֶּ 1256-15  # tav with dagesh
+always תַּ 1256-14  # tav with dagesh
+always תָּ 1256-126  # tav dagesh with qamats
+always בּ 12  # Bet with dagesh
+always בְּ 12-3  # Bet with dagesh sheva
+always כּ 13  # KAF with dagesh 
+always כְּ 13-3  # KAF with dagesh  sheva
+always תּ 1256  # tav with daggesh
+always תְּ 1256-3  # tav with daggesh sheva
+
+# letter op code characters
+letter ְ 3  # sheva
+letter ֱ 26  # hataf segol
+letter ֲ 25  # hataf patah
+letter ֳ 345  # hataf qamats
+letter ִ 24  # hiriq
+letter ֵ 34  # tsere
+letter ֶ 15  # segol
+letter ַ 14  # patah
+letter ָ 126  # qamats
+letter ֹ 135  # holam Malay
+letter ֺ 246  # holam haser
+letter ֻ 346  # qubuts
+letter ּ 5  # dagesh
+letter א 1  # alef
+letter ב 1236  # bet
+letter ג 1245  # gimel
+letter ד 145  # dalet
+letter ה 125  # hey
+letter ו 2456  # vav
+letter ז 1356  # zayin
+letter ח 1346  # het
+letter ט 2345  # tet
+letter כ 16  # kaf
+letter ל 123  # lamed
+letter מ 134  # mem
+letter ן 1345  # noback  final nun
```

Comment:
Here, noback is in the comment

Reply:
@LeonarddeR I went ahead and marked this "resolved" for you.